app/Http/Controllers/admin/adminController.php

```diff
@@ -0,0 +1,357 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Dat
+ * Date: 2/19/2019
+ * Time: 10:17 AM
+ */
+
+namespace App\Http\Controllers\admin;
+
+use App\Http\Controllers\Controller;
+use App\Level;
+use App\Users;
+use Illuminate\Http\Request;
+
+
+class adminController extends Controller
+{
+    public function show(Request $request) {
+        if (session('login')and !session('log'))
```

Comment:
góp ý sử dụng && thay cho "and"

Reply:
à, tại lúc đầu em thừ 1 dấu & k được ý, 2 cách này khác nhau gì hả a!

Reply:
giống nhau mà, nhưng a thấy hay sử dụng && trong các câu điều kiện, còn or and trong câu sql.